{{cookiecutter.project_slug}}/clients/mobile/react-native/babel.config.js

```diff
@@ -1,25 +1,6 @@
 module.exports = function (api) {
   api.cache(true)
   return {
-    presets: ['babel-preset-expo'],
-    plugins: [
-      'react-native-reanimated/plugin',
-      'nativewind/babel',
-      [
-        'module-resolver',
-        {
-          alias: {
-            '@components': './src/components',
-            '@screens': './src/screens',
-            '@stores': './src/stores',
-            '@assets': './assets',
-            '@services': './src/services',
-            '@utils': './src/utils',
-            '@src': './src',
-          },
-          extensions: ['.js', '.jsx', '.ts', '.tsx'],
-        },
-      ],
-    ],
+    presets: [['babel-preset-expo', { jsxImportSource: 'nativewind' }], 'nativewind/babel'],
```

Comment:
Expo 50 now simply reads the `tsconfig` file and makes this without the need of `module-resolver`